test/spec/data/dataset.js

```diff
@@ -296,6 +297,51 @@ module.exports = {
           }
         },
         {
+          name: 'destroy inside accumulate state',
+          beforeEach: function(){
+            var obj = new DataObject({
+                data: {
+                  hello: 'world'
+                }
+            });
+
+            var objDataset = new Dataset({
+                items: [
+                    obj
+                ]
+            });
+
+            resetEvents();
```

Comment:
We need this to reset event counter. This is needed to ensure that only one itemsChanged event was called after accumulate state off.

If we will omit this, we will get 2 itemsChanged events: first after init, second after accumulate state off. And IMHO this behavior is not evident